src/binding/python/openpmd_api/DataFrame.py

```diff
@@ -0,0 +1,65 @@
+"""
+This file is part of the openPMD-api.
+
+Copyright 2021 openPMD contributors
+Authors: Axel Huebl
+License: LGPLv3+
+"""
+import numpy as np
+try:
+    import pandas as pd
+    found_pandas = True
+except ImportError:
+    print("pandas NOT found. Install pandas for DataFrame support.")
```

Comment:
Do you have separation between critical errors and warnings? Why not to raise a custom exception here

Reply:
At this point I want to be able to not cause issues if pandas is not available but the function in this module is never called. I could also do this quietly, though, no reason to print here at all (throw is below).